content/en/docs/ops/configuration/security/migrate-alpha-policy/index.md

```diff
@@ -0,0 +1,338 @@
+
+| Major Differences | `v1alpha1` | `v1beta1` |
+|---------|------------------------|--------------------------------|
+| API stability | not backward compatible | backward compatible |
+| mTLS | `MeshPolicy` and `Policy` | `PeerAuthentication` |
+| JWT | `MeshPolicy` and `Policy` | `RequestAuthentication` |
+| Authorization | `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding` | `AuthorizationPolicy` |
+| Policy target | service name based | workload label selector based |
+| Port number | service ports | workload ports |
+
+Although `RequestAuthentication` in `v1beta1` security policy is similar to `v1alpha1` JWT policy, there is a notable
+semantics change. The `v1alpha1` JWT policy needs to be migrated to two `v1beta1` resources: `RequestAuthentication` and `AuthorizationPolicy`.
+
+This will change the JWT deny message due to the use of `AuthorizationPolicy`. In alpha version, the HTTP code 401 is returned
+with the body `Origin authentication failed`. In beta version, the HTTP code 403 is returned with the body `RBAC: access denied`.
+
+The `v1alpha1` JWT policy [`triggerRule` field](https://istio.io/v1.4/docs/reference/config/security/istio.authentication.v1alpha1/#Jwt-TriggerRule)
+is replaced by the `AuthorizationPolicy` with the exception that the [`regex` field](https://istio.io/v1.4/docs/reference/config/security/istio.authentication.v1alpha1/#StringMatch)
+is no longer supported.
+
+## Migration flow
+
+This section describes details of how to migrate the `v1alpha1` security policy workload.
+
+### Step 1: Find out related policies
+
+For each namespace, find out all `v1alpha1` security policies that has effect on workloads in the namespace. The result
+could include:
+
+- a single `MeshPolicy` that applies to all services in the mesh;
+- a single namespace level `Policy` that applies to all workloads in the namespace;
+- multiple service level `Policy` that applies to the selected services in the namespace;
+- a single `ClusterRbacConfig` that enables the RBAC on the whole namespace or some services in the namespace;
+- multiple namespace level `ServiceRole` and `ServiceRoleBinding` that applies to all services in the namespace;
+- multiple namespace level `ServiceRole` and `ServiceRoleBinding` that applies to the selected services in the namespace;
+
+### Step 2: Convert service name to label selector
+
+The `v1alpha1` policy selects target using service name, you should refer to the corresponding service definition to decide
+the label selector that should be used in the `v1beta1` policy.
+
+A single `v1alpha1` policy may include multiple services, this means it could be migrated to multiple `v1beta1` policies
+because the `v1beta1` policy currently only supports at most one label selector per policy.
+
+Also note the `v1alpha1` uses service port but the `v1beta1` uses the workload port, this means the port number might be
+different in the migrated `v1beta1` policy.
```

Comment:
How do we handle the service port vs workload port difference in the conversion tool?

Reply:
the tool will find and use the workload port from the service.